contracts/LifMarketMaker.sol

```diff
@@ -128,17 +143,15 @@ contract LifMarketMaker is Ownable {
       accumDistribution = accumDistribution.add(deltaDistribution);
 
       marketMakerPeriods.push(MarketMakerPeriod(
-        startBlockPeriod, startBlockPeriod.add(blocksPerPeriod).sub(1),
         deltaDistribution, accumDistribution
       ));
 
-      startBlockPeriod = startBlockPeriod.add(blocksPerPeriod);
     }
   }
 
   function getCurrentPeriodIndex() constant public returns(uint256) {
     require(block.number >= startBlock);
```

Comment:
I think the require has to change too, it should be `block.number.sub(totalPausedBlocks) >= startBlock`
Also, it should be an `assert`

Reply:
why? the call can be done at anytime after the marketMaker start, changing to asser yes, but I we dont need to change the condition

Reply:
I think it's more correct to discount the totalPausedBlocks to assert the current block is after "the actual" market maker start block. But we could also remove this require and just rely on the next instruction to throw. Whatever works for me. But if we keep it, it should be changed to an assert

Reply:
changed to assert